up-l1/zenoh.adoc

```diff
@@ -86,93 +96,72 @@ Different message types **MUST** use different Zenoh API.
 | CS6 | REAL_TIME
 |===
 
-=== URI Mapping
+==== TTL
 
-UUri **MUST** be able to be serialized into link:../basics/uri.adoc#42-micro-uris[micro form].
-Then we can transform the micro-form UUri, which is `u8` array, into string in lowercase hexadecimal representation.
+The TTL only takes effect while sending RPC in the Zenoh implementation.
+TTL **MUST** be mapped to the timeout configuration in Zenoh query.
```

Comment:
what about other messages that have timed out already on arrival? Would you still deliver those to registered listeners?

Reply:
Maybe I missed something, but is there any way for us to know when the message is sent? How does the receiver side know the message is timed out only with TTL?

Reply:
The UUIDv8 defined in the specifications includes the timestamp from when the request message was generated.

Reply:
OK, I missed that.

> what about other messages that have timed out already on arrival? Would you still deliver those to registered listeners? Or send out messages in send() that have already timed out?

Then we should check whether messages time out or not on arrival in the transport implementation. Let me update it.

Reply:
Hmm... do you think we should drop the expired messages on the transport layer (Zenoh, MQTT..), or just passed to users and leave the decisions to them?